zulip/integrations/bridge_with_slack/run-slack-bridge

```diff
@@ -142,7 +142,8 @@ if __name__ == "__main__":
         exit(1)
 
     print("Starting slack mirroring bot")
-    print("MAKE SURE THE BOT IS SUBSCRIBED TO THE RELEVANT ZULIP STREAM")
+    print("MAKE SURE THE BOT IS SUBSCRIBED TO THE RELEVANT ZULIP STREAM(S)!")
+    print("MAKE SURE THE BOT IS SUBSCRIBED TO THE RELEVANT SLACK CHANNEL(S)!")
```

Comment:
Good to include, but I think as a user, seeing these two lines 'shouting' at me with a lot of common text, I might not notice that they're subtly different. Maybe an (expanded) formatting on three lines relating to 1) common subscribe text 2) zulip stream(s) 3) slack channel(s) ?

Reply:
I have combined the 2 lines into 1 sentence.